components/command/node/jest-config.mjs

```diff
@@ -42,7 +42,9 @@ const loadPackageJsonAsync = async (directory) => {
       return null;
     } else {
       logError("Could not load package.json from %j >> %O", directory, error);
-      throw new ExternalAppmapError("Could not load package.json");
+      throw new ExternalAppmapError(
+        format("Could not load package.json (%O)", [error]),
```

Comment:
The problem is, turning logging on is another step which can cause friction and it provides a lot of potentially unnecessary information. I don't want to have to rerun CI and delve through all the log messages to figure out what the actual single error was. Current laconic error messages give the user zero indication of what actually went wrong. We don't need (and we shouldn't) give the user a stacktrace by default (although we actually do!), but I believe we should (and need to) at least tell the user more specifically want went wrong instead of a enigmatic generic message that leaves them puzzled and confused.